app/controllers/tweet_controller.rb

```diff
@@ -0,0 +1,84 @@
+require 'pry'
+
+class TweetController < ApplicationController
+
+  get '/tweets' do
+
+    if !logged_in?
+      redirect to '/login'
+    end
+    @user = User.find_by_id(session[:user_id])
+    erb :'tweets/show'
+  end
+
+  get '/tweets/new' do
+    if logged_in?
+      erb :'tweets/new'
+    else
+      redirect to '/login'
+    end
+  end
+
+  post '/tweets' do
+    if (params[:tweet][:content]).empty?
+      redirect to '/tweets/new'
+    else
+      @user = User.find_by_id(session[:user_id])
+      @tweet = Tweet.create(params[:tweet])
+      @user.tweets << @tweet
+      redirect to '/tweets'
+    end
+  end
+
+  get '/tweets/:id' do
+    if logged_in?
+      @user = User.find_by_id(session[:user_id])
+      @tweet = Tweet.find_by_id(params[:id])
+          erb :'tweets/show_individual_tweet'
+    else
+      redirect to '/login'
+    end
+  end
+
+  get '/tweets/:id/edit' do
+    if logged_in?
```

Comment:
Shouldn't you verify a tweet belongs to the `current_user` (and not just that any user is logged in) before allowing them to edit?